### ISSUE TITLE
bug: Manager showing up uninstalled app in Installed list

### ISSUE THREAD

**Issue by ViktorVoloshko** [revanced/revanced-manager#119]
### Type

Cosmetic

### Branch

Flutter

### Bug description

When I patched app, installed it and then deleted info about app still available in Installed list.

### Steps to reproduce

1) Patch app
2) Install it
3) Delete it
4) Check patched and installed apps on Manager

### Relevant log output

```shell
Make this optional plz. :)

Instead create app info requirement or optional. Here's mine:
Version: 0.0.7
Model: M2010J19CG
Android Version: 12
Arch: arm64-v8a
```

### Screenshots or videos

![Screenshot_20220916-090225_ReVanced Manager](https://user-images.githubusercontent.com/87974065/190568243-0d9a562c-b34f-49f4-8b69-6e9b2081f97c.png)
![Screenshot_20220916-090229_ReVanced Manager](https://user-images.githubusercontent.com/87974065/190568254-b7877d5b-f7ad-42c3-8114-9d66d383bbcf.png)
![Screenshot_20220916-090256_Settings](https://user-images.githubusercontent.com/87974065/190568265-420b8fe3-e1b0-4c9d-a604-3b130d0cc777.png)

**Comment by E85Addict** [comment]
Too add to this, the uninstall button is not working in the manager for me and doing the same as the bug he is having.

**Comment by ponces** [comment]
Fixed on v0.0.12.
Closed.
